plutus-pab/src/Plutus/PAB/Webserver/Server.hs

```diff
@@ -112,8 +114,12 @@ startServer ::
     -> Either ClientEnv (PABAction t env WalletInfo)
     -> Availability
```

Comment:
Why not `[Middleware]` here?  Users can then call this function and set up their server accordingly.